performance-tests/selectors.mjs

```diff
@@ -0,0 +1,23 @@
+// @flow strict
+
+const selectors = {
```

Comment:
Do these need to be repeated from `/playwright/masonry/utils`? Can we use those selectors instead of duplicating them here?